### PR TITLE
fix: fix data binding on news preview

### PR DESCRIPTION
**Pull request by bangunbagustapa**
### Changes

- Fix data binding on news preview

### Evidence

Title: fix: fix data binding on news preview
Project: Portal Jabar
Participants: @bangunbagustapa @Ibwedagama @yoslie

### CODE REVIEW

File: src/pages/News/CreateEditNews.vue [hunk 1]
```diff
@@ -741,7 +740,11 @@ export default {
           endDate: data.end_date ? formatDate(data.end_date, 'dd/MM/yyyy') : null,
           category: data.category,
           tags: data.tags,
+          author: data.author,
+          reporter: data.reporter,
+          editor: data.editor,
           areaId: data.area.id,
+          updated_at: data.updated_at,
```
Comment (Ibwedagama) [inline src/pages/News/CreateEditNews.vue:747]: hello @bangunbagustapa , If the user still creating the news, the `updated_at` still empty doesn't it?  🤔

**Review by Ibwedagama — APPROVED** [review]

File: src/pages/News/CreateEditNews.vue (hunk 1 above)
Comment (bangunbagustapa) [reply src/pages/News/CreateEditNews.vue:747]: Well, it is. I think the `NewsPreview` component should have a default value like the current date? Wdyt?

Comment (Ibwedagama) [reply src/pages/News/CreateEditNews.vue:747]: yes I think it's better to handle it on the `NewsPreview` component

Comment (bangunbagustapa) [reply src/pages/News/CreateEditNews.vue:747]: Alright, changes here https://github.com/jabardigitalservice/portal-jabar-cms/pull/196/commits/c22d9b86f743a2620888f92f54097d98df56d3f9. Thank you!

**Review by yoslie — APPROVED** [review]

**Pull request merged**